client/scripts/models/Profile.ts

```diff
@@ -80,7 +80,7 @@ class Profile {
   // or just the address if the user has not set a name.
   get displayName() : string {
     if (!this._initialized) return 'Loading...';
-    return this.name || formatAddressShort(this.address, this.chain);
+    return this.name || 'Anonymous';
```

Comment:
Are we sure we want this change?

Reply:
We use `displayNameWithAddress()` to show both, this is just for setting the `this.name` value. This is a valid change.